src/sentry/api/helpers/group_index/index.py

```diff
@@ -80,7 +80,15 @@ def build_query_params_from_request(
         except ValueError:
             raise ParseError(detail="Invalid cursor parameter.")
     has_query = request.GET.get("query")
-    query = request.GET.get("query", "is:unresolved").strip()
+    query = request.GET.get("query", None)
+    if query is None:
```

Comment:
We should probably also handle an empty string query here

Reply:
Actually @snigdhas I think the existing behavior was correct? We still need to allow users to search without any filters, which would be the empty search string

Reply:
lol smart 🧠